src/Integration.Vsix/Settings/GeneralOptionsDialogControl.xaml.cs

```diff
@@ -42,23 +45,35 @@ public GeneralOptionsDialogControl(ISonarLintSettings settings, ISonarLintDaemon
             {
                 throw new ArgumentNullException(nameof(daemon));
             }
+            if (logger == null)
+            {
+                throw new ArgumentNullException(nameof(logger));
+            }
 
             this.settings = settings;
             this.daemon = daemon;
+            this.logger = logger;
 
             InitializeComponent();
         }
 
         protected override void OnInitialized(EventArgs e)
         {
-            base.OnInitialized(e);
+            try
+            {
+                base.OnInitialized(e);
 
-            if (!daemon.IsInstalled)
+                if (!daemon.IsInstalled)
+                {
+                    settings.IsActivateMoreEnabled = false;
+                }
+
+                UpdateActiveMoreControls();
+            }
+            catch (Exception ex) when (!ErrorHandler.IsCriticalException(ex))
```

Comment:
I have the nasty feeling that all of these lines of code will be treated as changed and that the quality gate will break as a result. Fingers crossed it doesn't, as none of this code is easy to test.